db/seeds.rb

```diff
@@ -1,10 +1,11 @@
+# ruby encoding: utf-8
 require 'nokogiri'
 require 'open-uri'
 
-page = Nokogiri::HTML(open("https://quizlet.com/50590/1000-top-used-spanish-words-flash-cards/"))
+page = Nokogiri::HTML(open("http://masterrussian.com/vocabulary/most_common_words.htm"))
 
-page.css('.term').each do |words|
-  unless (words.css(".lang-es")[0]).nil? || (words.css(".lang-en")[0]).nil?
-    Card.create!( original_text: words.css(".lang-es")[0].content, translated_text:  words.css(".lang-en")[0].content)
+page.css('tr').each do |words|
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.